### ISSUE TITLE
Get updates of the science domain pages after their WP4 presentations

### ISSUE THREAD

**Issue by lauportell** [EVERSE-ResearchSoftware/RSQKit#121]
Update the science domain page with the information from the presentation done by all science clusters in the WP4 meetings

**Comment by juckel** [comment]
@lauportell thanks a lot for the reminder. Caterina and I brainstormed a bit today and are wondering, if instead of lengthening the science cluster pages, we ask the WP4 members to create "science story" pages (in a sense similar to the "your task" pages) for each pilot (and also follow up science use case). From the discussion with @gperu about what to do during the de-RSE RSQkit contentathon, he mentioned the idea of science case pages as well.

If you believe this to be beneficial as well, could you please point me to the science use case template so that we can get started on this. Thanks!

**Comment by lauportell** [comment]
Hi @juckel,
I think its a great idea to populate the science cluster pages with a "science story" as you propose, to make similar to the task pages. If the rest of the editorial board agrees., I think we can get started on this.

@anenadic @shoaibsufi @jmaassen @egpbos @dgarijo what do you think?

**Comment by shoaibsufi** [comment]
The RDMKit by comparison has - https://rdmkit.elixir-europe.org/your_domain pages on domain specific best practices for data management - we have not thought of a strong analogue for RSQKit - as there is a lot of domain specificity around data types and how to handle and manage them - but in research software - most of the quality characteristics are similar for each domain (some examples of domain specific advice maybe - in algorithm selection from problem types or the most appropriate libraries/infrastructure to use e.g. Bioconductor for bioinformatics and Astropy for astronomy).

I am not sure what "science stories" would be about - the crux of the issue is the 'science use case template' - that's a discussion in itself - what should be in it - at the moment the pages under https://everse.software/RSQKit/research_communities describe some pilot use cases - but only touch on quality.

@juckel how do you see these stories linking to software quality? I think if we could see a draft of one then it would become clearer what you had in mind (and also a template might come from that).

On another note - I certainly think more work needs to be done to bring in the pilots to understand:

 - What aspects of software quality they are doing well - and could therefore e.g. write task page content about.
- Where they want to go next in terms of quality - and therefore what task page content they would like to see

**Comment by juckel** [comment]
Thanks all for your patience with me not responding quickly. My main rationale was to decouple the cluster introduction from the pilots. The idea is that the pilots serve as examples of codes or software practices from the science clusters that both represent the clusters but were also chosen in the project proposal phase to complement each other across the research software space to provide a potentially very broad picture of what research software is and does. Some clusters (like Elixir) have already extended their pilots. Hence I am looking for a better mechanism to present the pilots (and potential later use case stories) following the scheme we developed in WP4 about what we asked all members about the pilots. This lead me to the idea of "science story" pages or rather use case pages of essentially turning the presentations from our WP4 into a resource for the public to see. I am happy with any other format you suggest for this, however.

I also agree with your next steps - as this was also something I raised in the WP6 meeting about WP4 folks now waiting for "things to do" to provide feedback on the outcomes of the other WPs.

**Comment by shoaibsufi** [comment]
@juckel can you link to or remind what the 'scheme we developed in WP4 about what we asked all members about the pilots' was - that might be a great start for a discussion about the template for these 'software story' pages.

**Comment by juckel** [comment]
yes, you are right. I could have added this in the fist place. here is what we asked all the pilots:

## We would ask you to have some slides (<20-25’ in total) for all following points

* These points have been inspired by the TF2 reference model draft [here](https://warehouse.inab.certh.gr/index.php/apps/onlyoffice/3754561?filePath=%2FMyResearch%2F04.%20Projects%2FEU-Funded%2FEVERSE%20(HORIZON-INFRA-2023-EOSC-01-02)%2FConsortium%2F2.WPs%2FTF2-ReferenceModel%2FTF2_reference_model.docx) (OwnCloud link) 
* We can provide a google slides template if you want to use it

1. Describe the community that uses the software in your pilot 
   * Who is the developer? (One, many, organised, less organised, changeable…)
   * Who are the users, and how experienced are they? 
2. Describe the technical aspects of the code in your pilot(s) according to the following template:
   * Software tier 
      * Tier 1: Analysis code
      * Tier 2: Prototype Tools
      *  Tier 3: Research Software Infrastructure (includes services)
   * Technical specifications: language(s), size of codebase, where does it run (specify if it’s on a VRE, HPC…)   
3. Describe your software lifecycle [https://zenodo.org/records/8324828]
   * This includes information in (1) but also the following points:
      * How has it been developed so far (version control, contribution policy)
      * How do you onboard new developers/users? 
   * You can start by describing a “user story” (see page 4 of reference model)
4. Are there indicators and tools applicable to your pilot(s) that provide insight into technical quality? 
   * Do you or others in your community use them? 
5. Describe the FAIR / Open aspects of the code in your pilot(s):
   * See https://fair-software.eu/ for inspiration: are there a repository and a license, do you have a community registry/catalogue where the code is available, is it citeable, do you have a “good software” checklist 
   * How is the documentation maintained / by whom
6. Does your software have a sustainability plan?
   * This includes information on management, funding, contributors
7.   In your community, do you already have/know of training resources for developing/using software? 
   * WP5 has a list of resources [here](https://docs.google.com/spreadsheets/d/1AZgDJf0UI96UZv9UCTIbMg6J6a0sUFSXMq3ep3XHdoM/edit#gid=0), we’ll contribute to that if anything is not listed already 
8. How are software developers rewarded in your community?

**Comment by shoaibsufi** [comment]
I am closing this issue as we now have the template as part of https://github.com/EVERSE-ResearchSoftware/RSQKit/pull/234 

And replacing it with the following two tickets for the outstanding research community entries (we can change the section to research/science story at a later time if that's better - currently keeping research community).

- https://github.com/EVERSE-ResearchSoftware/RSQKit/issues/237
- https://github.com/EVERSE-ResearchSoftware/RSQKit/issues/236
